pypodio2/api.py

```diff
@@ -16,9 +16,15 @@ def OAuthClient(api_key, api_secret, login, password, user_agent=None,
     return AuthorizingClient(domain, auth, user_agent=user_agent)
 
 
+def OAuthRefreshTokenClient(client_id, client_secret, refresh_token, user_agent=None,
```

Comment:
Is there an special reason to use CamelCase for this and other functions' names?

Reply:
I wouldn't use CamelCase personally, just did so to follow the project coding style.

Furthermore this fork is public so I think it would be best to only extend and not break to much. This way if anybody else wants to use this fork they can easily do so without having to change their existing code from the original version.

Reply:
I see, I wasn't aware this was a fork. Makes sense.